backend/routes/interactions.js

```diff
@@ -103,9 +110,9 @@ router.put("/update-scores", async (req, res) => {
         where: { newsId: interaction.newsId },
       });
 
-      const weightedLiked = interaction.isLiked ? 3.0 : 1.0;
-      const weightedOpen = interaction.openCount * 3; // 3 times whatever the number was
-      const weightedRead = interaction.readCount * 3;
+      const weightedLiked = interaction.isLiked ? WEIGHTS.LIKED : 1.0;
```

Comment:
These numbers are probably somewhat arbitrary, but why do we give something that isn't liked a weight of 1?